### PR TITLE
Add range slider

### PR DESCRIPTION
**Pull request by maya**
This adds the range slider to the library using the HTML5 range input.

This addresses #244 

This is what it looks like:
screen shot 2015-08-03 at 4 04 06 pm

Live :movie_camera: 

Note: range inputs will degrade to text inputs in IE9 and below.

### CODE REVIEW

File: assets/_scss/elements/_inputs.scss [hunk 1]
```diff
@@ -138,3 +138,98 @@ input[type="radio"]:disabled + label::before {
...
+  border: 1.8px solid $color-grey-600;
+  height: 20px;
+  border-radius: 15px;
+  background: #eeeeee;
+  cursor: pointer;
+  margin-top: -6.5px;  
+  width: 20px;
+}
+
+input[type=range]:focus::-webkit-slider-runnable-track {
```
Comment (msecret) [inline assets/_scss/elements/_inputs.scss]: Are we going to be able to replace these with generics when we add auto-prefixer? Or are these stuck as browser prefixes?

Comment (maya) [inline assets/_scss/elements/_inputs.scss]: I'm not positive how autoprefixer works with selectors, I know it'll be able to replace the attributes.

Comment (msecret) [inline assets/_scss/elements/_inputs.scss]: Do you know what browsers this is going to work on?

Comment (maya) [inline assets/_scss/elements/_inputs.scss]: http://caniuse.com/#feat=input-range

File: assets/_scss/elements/_inputs.scss [hunk 2]
```diff
@@ -139,3 +139,97 @@ input[type="radio"]:disabled + label::before {
...
+  height: 2rem;
+  border-radius: 1.5rem;
+  background: #eeeeee;
+  cursor: pointer;
+  margin-top: -.65rem;  
+  width: 2rem;
+}
+
+input[type=range]::-moz-range-thumb {
+  background: #eeeeee;
```
Comment (msecret) [inline assets/_scss/elements/_inputs.scss]: Why aren't these background values using variables?

Comment (maya) [inline assets/_scss/elements/_inputs.scss]: Fixed in 9a1f8a6.

**Pull request merged**